webpack.server.config.js

```diff
@@ -6,52 +6,57 @@
  *
  * Note: that prod and dev mode are set in npm scripts.
  */
-const path = require("path")
-const nodeExternals = require("webpack-node-externals")
-const NodemonPlugin = require("nodemon-webpack-plugin")
-const webpack = require("webpack")
+const path = require('path')
+const nodeExternals = require('webpack-node-externals')
+const NodemonPlugin = require('nodemon-webpack-plugin')
+const SentryCliPlugin = require('@sentry/webpack-plugin')
+const webpack = require('webpack')
 module.exports = (env, argv) => {
-  const SERVER_PATH = (argv.mode === "production")
-    ? "./src/server/index-prod.js"
-    : "./src/server/server-dev.js"
+  const SERVER_PATH = argv.mode === 'production' ? './src/server/index-prod.js' : './src/server/server-dev.js'
 
-  return ({
+  return {
     entry: {
-      server: ["@babel/polyfill",SERVER_PATH]
+      server: ['@babel/polyfill', SERVER_PATH]
     },
     output: {
-      path: path.join(__dirname, "dist"),
-      publicPath: "/",
-      filename: "[name].js"
+      path: path.join(__dirname, 'dist'),
+      publicPath: '/',
+      filename: '[name].js'
     },
     mode: argv.mode,
     devtool: '#source-map',
-    target: "node",
+    target: 'node',
     node: {
       // Need this when working with express, otherwise the build fails
       __dirname: false, // if you don't put this is, __dirname
-      __filename: false, // and __filename return blank or /
+      __filename: false // and __filename return blank or /
     },
     optimization: {
-    nodeEnv: false
-  },
+      nodeEnv: false
+    },
     externals: [nodeExternals()], // Need this to avoid error when working with Express
     plugins: [
-       new webpack.DefinePlugin({}),
-      new NodemonPlugin()],
+      new webpack.DefinePlugin({}),
+      new NodemonPlugin(),
+      new SentryCliPlugin({
```

Comment:
according to the docs ignoreFile isn't and .gitignore, it's a Sentry-specific .sentrycliignore file
also you should provide path to sentry.properties which identifies you in the system (your account your license etc)

Reply:
please, read:
https://docs.sentry.io/cli/configuration/
https://docs.sentry.io/cli/releases/#sentry-cli-sourcemaps

Reply:
I already read it. We can just pass required env variables which sentry plugin will accept automatically

from docs:
> can be configured with a config file named .sentryclirc as well as environment variables 

Regarding .gitignore - see plugin docs (ignoreFile option) - https://github.com/getsentry/sentry-webpack-plugin
it says that we can point to .gitignore